FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -656,6 +697,11 @@ void prvDefenderDemoTask( void * pvParameters )
 
     /*********************** Collect device metrics. **********************/
 
+    /* We then need to collect the metrics that will be sent to the AWS IoT
+     * Device Defender service. This demo puts the functions for collecting
+     * metrics in metrics_collector.h, as to abstract over the transport
+     * implementation. This demo provides metrics_collecter.c which implements
+     * the demo's metric collection for the FreeRTOS+TCP stack. */
```

Comment:
Suggestion for rewording.
/* We then need to collect the metrics that will be sent to the AWS IoT
 * Device Defender service. This demo uses the functions declared in
 * in metrics_collector.h to collect network metrics. For this demo, the
 * implementation of these functions are in metrics_collector.c and
 * collects metrics using tcp_netstat utility for FreeRTOS+TCP. */